Police malpractice (#8)
chizarlicious/chizarlicious.github.io: _posts/2017-08-21-police-malpractice.md

Doctors make life-and-death decisions millions of times each year. Sometimes they get it wrong, and people die. Unnecessary surgery, incorrect prescriptions, and other medical errors account for over [100k American lives lost](http://www.npr.org/sections/health-shots/2013/09/20/224507654/how-many-die-from-medical-mistakes-in-u-s-hospitals) each year -- more than car accidents and guns combined. However, despite how often Americans[^1] die from medical errors, it's exceptionally rare for a doctor to face criminal charges for murder or manslaughter. Instead, victims typically seek justice in civil court for [medical malpractice](https://en.wikipedia.org/wiki/Medical_malpractice_in_the_United_States).

[^1]: Medical errors aren't a uniquely American phenomenon. [Canada](http://globalnews.ca/news/3026275/1-in-18-canadian-hospital-patients-experience-harm-from-preventable-errors-study/), the [UK](https://www.theguardian.com/society/2015/jul/14/avoidable-deaths-nhs-hospitals-study), and [Germany](http://www.upi.com/17000-die-in-Germany-from-medical-errors/52721267847072/) have similar rates.

> The medical malpractice system is by no means perfect. It's [expensive](http://doctorly.org/cost-vs-reward-of-opening-a-medical-private-practice/), [bureaucratic](http://www.nejm.org/doi/full/10.1056/NEJMsa054479), and many suits are [frivolous](https://www.irmi.com/articles/expert-commentary/medical-malpractice-the-high-cost-of-meritless-claims). But it gets the job done. Doctors and nurses are [among the most trusted](https://news.gallup.com/poll/274673/nurses-continue-rate-highest-honesty-ethics.aspx) professions, and the cost of malpractice insurance puts them under constant [pressure to improve](http://m.hartfordbusiness.com/article/20140428/PRINTEDITION/304249935/hospitals-battle-medical-malpractice-costs) further. High risk means high insurance premiums, so hospitals are saving money by reducing risk. They're [training staff](http://www.philly.com/philly/business/20151122_Medmal_head_here_in_two_lines_of_36_pt.html) to reduce medical errors -- and studies show these [trainings are effective](https://www.ncbi.nlm.nih.gov/pubmed/27549442).

Reviewer: I like "They train" more than "they are training".

> If you go to the gym, your health insurance gets cheaper. If you replace your brakes, your car insurance gets cheaper. If you fix your roof, your homeowner's insurance gets cheaper. The insurance company creates incentives for responsible long-term behavior, and it's not because they care about your health or safety. They make their money by identifying liabilities and putting a price tag on them.

Reviewer: > They make their money by identifying liabilities and putting a price tag on them.

Well put.

[^2]: In Minnesota alone, police paid out $1.8M after killing [Dominic Felder](https://www.mprnews.org/story/2010/10/25/excessive-force-verdict) in 2006, $200k after killing [Jamal Clark](https://www.mprnews.org/story/2019/08/09/attorneys-for-jamar-clark-family-members-confirm-agreement-on-200000-settlement) in 2014, and $3M after killing [Philando Castile](https://en.wikipedia.org/wiki/Shooting_of_Philando_Castile) in 2017.

One path forward is to require each officer to hold liability insurance, rather than being covered by the department or city. The officers who killed George Floyd were [repeatedly reprimanded](https://www.mprnews.org/story/2015/11/19/the-officers-in-the-jamar-clark-shooting) for excessive force. The officers who killed Jamal Clark had a [history of violence](https://www.mprnews.org/story/2015/11/19/the-officers-in-the-jamar-clark-shooting) as well. Each of these men was a multimillion-dollar settlement waiting to happen, which any insurance underwriter could have seen. Prohibitive insurance costs should have made them unemployable as police officers long before they had a chance to kill, regardless of support from the department or union.

> <!-- the cost of insurance could also be shared by superior officers, creating an incentive not to hire high-risk officers. but that's a next-order effect, and explaining might be more of a distraction than it's worth -->

Reviewer: Agree probably not worth it.

> Addressing police misconduct through civil suits and liability insurance may also help shift public opinion. Under [today's laws](https://en.wikipedia.org/wiki/Qualified_immunity), it's [hard to convict the police](https://www.politico.com/news/2020/06/01/minnesota-attorney-general-hard-to-convict-police-293920). They're [rarely even indicted](https://fivethirtyeight.com/datalab/ferguson-michael-brown-indictment-darren-wilson/). But criminal cases attract far more attention than civil suits (see chart below). A [consistent majority](http://www.gallup.com/poll/213869/confidence-police-back-historical-average.aspx) of Americans report a great deal of confidence in the police. To advance reforms at the ballot box, that confidence needs to be shaken. Putting million-dollar settlements front and center might do the trick. "[Officer who shot Philando Castile found not guilty on all counts](http://www.cnn.com/2017/06/16/us/philando-castile-trial-verdict/index.html)" won't.

Reviewer: Let's try to draw out the logic, because I don't think it flows well right now.

> People ignore civil suits -> People Like police -> ??? -> People don't like police

I think you are trying to argue: People currently like police, partially because they ignore civil law suits. But if we made criminal lawsuits not possible, and civil more common/large, people would pay attention to those.

I think you have some unstated parts of your argument (like if there are no criminal suits), and I don't know if I agree with that conclusion.

<!-- this wording still isn't quite right. would like to note that this is the *climactic* headline. the one thing that everyone reads.  -->

![In terms of search traffic, the climax of the Philando Castile case was Officer Yanez's acquittal, not the $3M wrongful death settlement.](/assets/images/wide/malpractice-castile-trends-16x9.png)
*In terms of [search traffic](https://g.co/trends/9eJeo), the climax of the Philando Castile case was Officer Yanez's acquittal. The family's $3M wrongful death settlement with the city received comparably little attention.*

> In the healthcare industry, malpractice insurance puts a price tag on risk. High-risk doctors are more expensive to employ, and hospitals save money by training staff to avoid mistakes. Requiring police officers to hold liability insurance would work similarly, saving taxpayer dollars and decreasing the use of excessive force.

Reviewer: ```suggestion
In the healthcare industry, malpractice insurance puts a price tag on risk. High-risk doctors are more expensive to employ, and hospitals save money by training staff to avoid mistakes. This increases the public's trust in doctors. Requiring police officers to hold liability insurance would have similar effects; it would save taxpayer dollars and decrease the use of excessive force.
```

Author: Updated to call out public trust earlier. Not sure it's central enough to go in the conclusion

![Noodle's first protest](/assets/images/blm.png)
*[George Floyd](https://en.wikipedia.org/wiki/Death_of_George_Floyd) was killed in police custody less than a mile from our home.*

> American police departments pay out [hundreds of millions of dollars](https://www.wsj.com/articles/cost-of-police-misconduct-cases-soars-in-big-u-s-cities-1437013834) each year due to police misconduct, including wrongful deaths at the hands of police[^2]. Clearly, getting liabilities under control could save taxpayers quite a bit of money. But that's just the thing: because police departments are publicly funded, there are no shareholders to push back against reckless spending. Police may even piggyback on the [city's insurance](http://www.savmn.com/DocumentCenter/View/568/6-26-17--Joint-Statement-City-of-St-Anthony-and-Valerie-Castile-as-Trustee-for-Philando-Castile) policy, further insulating them from the costs they incur.

Reviewer: > there are no shareholders to push back against reckless spending.

What about voters?

Author: "to push back in real time"?

voters don't have a direct financial stake in government efficiency, but maybe response time is clearer as a barrier to feedback
